terminus-ui/form-field/src/form-field.component.scss

```diff
@@ -363,7 +378,9 @@ $outline-appearance-label-offset: -.25em;
   .ts-form-field {
     // Center the datepicker icon vertically
     .ts-form-field__suffix {
-      padding: .5em 0;
+      $padding-adjustment-for-datepicker: .5em 0;
```

Comment:
What's the rhyme and reason for moving values to variables? In other words, why was `padding` moved to a variable, but `top` was not?

Reply:
We have a rule specifically looking for 'random' values being used in the `padding` or `margin` declarations since we know for sure that any dev should be using our `spacing` helper. We aren't enforcing that for every rule because there will be plenty of edge cases where someone may need to adjust `top` etc for visual alignment that doesn't fit our spacing helper

Reply:
It's also a bit like the JS rule `no-magic-numbers`.. forcing the dev to put the number in a variable makes them specify _why_ it's needed